src/docs/Foundations/MagicUnit/index.mdx

```diff
@@ -8,20 +8,20 @@ status:
 
 > **The magic unit is a base unit for any element or property that require a defined dimension**. It is equal to **16px** but is expressed in **rem**.<br /> We multiply or divide it to define white spaces, icons sizes, lines heights etc...
 
-## Why do we use the magic unit ?
+## Why do we use the magic unit?
 
 Using a magic unit have multiples benefits.
```

Comment:
~multiples~ multiple. 
Not sure about this :/

Reply:
fixed